### PR TITLE
Added remove preview image button

### PR DESCRIPTION
**Pull request by brfeitoza**
Hi matheusfelipeog, sorry for duplicated.

Now i'm created another PL with just the new button to remove preview image.

I hope it's useful.

### CODE REVIEW

**Comment by matheusfelipeog** [comment]
Very good, @brunoff634cyt 😆 

You removed only one thing. I will open a code review for you to see and resolve the issue, ok?

After that, I accept the PR for merge.

**OBS:** This PR is valid for [hacktoberfest](https://hacktoberfest.digitalocean.com/), if you are interested in obtaining a shirt of event 😄

File: web/src/pages/CreateOrphanage.tsx [hunk 1]
```diff
@@ -142,10 +166,9 @@ export default function CreateOrphanage() {
 
               <input
                 type="file"
-                id="image[]"
                 multiple
-                accept=".png, .jpg, .jpeg"
```
Comment (matheusfelipeog) [inline web/src/pages/CreateOrphanage.tsx:147]: Do not remove the `accept` property, it is used for image type limiting

**Review by matheusfelipeog — CHANGES_REQUESTED** [review]

File: web/src/pages/CreateOrphanage.tsx (hunk 1 above)
Comment (brfeitoza) [reply web/src/pages/CreateOrphanage.tsx:147]: I got it, very good. 

I made a new commit with the change. 

Thanks Matheus 😄

**Comment by matheusfelipeog** [comment]
Very good, now it's ok, @brunoff634cyt 🎉

Thanks for the contribution :)

**Review by matheusfelipeog — APPROVED** [review]

**Pull request merged**